src/Language/Fixpoint/Solver/Monad.hs

```diff
@@ -67,30 +64,11 @@ data SolverState = SS
   , ssStats   :: !Stats            -- ^ Solver Statistics
   }
 
-data Stats = Stats 
```

Comment:
We moved this `Stats` data structure into a standalone module to avoid cyclic imports, as this is now used inside `FixResult`.